lib/compiler.js

```diff
@@ -8,6 +8,7 @@ var debug = require('debug')('loopback:boot:compiler');
 var Module = require('module');
 var _ = require('lodash');
 
+var FILE_EXTENSION_JSON = '.json';
```

Comment:
Nitpick: please add an empty line to visually delimit the variable (constant) declaration from the next function.